zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestFederatedZkClient.java

```diff
@@ -662,4 +654,124 @@ public void testClose() {
       Assert.assertEquals(ex.getMessage(), "FederatedZkClient is closed!");
     }
   }
+
+  /**
+   * Test that zk multi works for op.create.
+   */
```

Comment:
I think since this test class already extends RealmAwareZkClientTestBase, the following tests should already be included. 
We could add test cases that are specific to federatedZKClient implementation of multi. For example, try to create multiple ZNode that do not belongs to one realm and expect a failure.